plugin/src/test/java/io/jenkins/plugins/analysis/core/util/ModelValidationTest.java

```diff
@@ -44,7 +44,7 @@ void shouldRejectId(final String id) {
     }
 
     @ParameterizedTest(name = "{index} => Should be marked as valid ID: \"{0}\"")
-    @ValueSource(strings = {"", "a", "awordb", "a-b", "a_b", "", "0", "a0b", "12a34", "A", "aBc", "a.b-c_e"})
+    @ValueSource(strings = {"", "a", "awordb", "a-b", "a_b", "", "0", "a0b", "12a34", "A", "aBc", "a.b-c_e", "z.."})
```

Comment:
Win32 API generally removes trailing dots from file names, but that's OK because:

- If Warnings NG always adds a suffix, then the dots won't be trailing and won't be removed by Win32.
- AFAIK the plugin does not implement any denylist that a user could circumvent by adding such dots.